lib/requests/RequestActions.js

```diff
@@ -10,6 +10,7 @@ class RequestActions {
     } else {
       studyRequest.status = StudyRequestStatus.ASSIGNED;
     }
+    studyRequest.closed = false;
```

Comment:
at this point, the `closed` flag doesn't really mean anything - we can infer it from `status`.  (don't need to remove it yet, but might be worth moving in that direction over the next couple of releases.)